lms/djangoapps/courseware/views/views.py

```diff
@@ -1588,6 +1588,13 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
         student_view_context = request.GET.dict()
         student_view_context['show_bookmark_button'] = False
 
+        # xBlocks iframed into the learning microfrontend application will
+        # not render some parts of an xblock template (notably unit_title
+        # in vert_module.html). The variable name is_microfrontend_embed
+        # is used instead of adding specific flags like show_unit_title to
+        # help keep track of why certain features may be rendered or not.
+        student_view_context['is_microfrontend_embed'] = bool(request.GET.get('is_microfrontend_embed', False))
```

Comment:
This would evaluate to true if `is_microfrontend_embed=false` or `is_microfrontend_embed=pizza`. Is that the intention? 

Otherwise, maybe you want `student_view_context['is_microfrontend_embed'] = request.GET.get('is_microfrontend_embed', '') == 'true'`

Reply:
Do we have an existing pattern for booleans in query strings?

Reply:
I found `include_language_selector = request.GET.get('include-language-selector', '') == '1'` here, but it's the only other example of this I found in the platform. https://github.com/edx/edx-platform/blob/436122063824d5fe9efb6286e9ba304713e3ff36/lms/djangoapps/branding/views.py#L267